addon/components/ownership-selection-modal.js

```diff
@@ -15,19 +19,42 @@ export default Component.extend({
   header: 'Share with',
   cta: 'Share',
 
+  items: computed('searchTerm', function() {
+    let members;
+
+    return this.currentUser.fetchColleagues().then((res) => {
```

Comment:
Given that you're not using another key of `res` in the rest of the code, you can pass `({ users }) => {}` instead of `(res) => {}`. This will give you direct access to the `users` key of the Promise resolution (here `res`) (cf. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring)

Reply:
It's actually the principle that you used on https://github.com/upfluence/ember-upf-utils/pull/189/files#diff-dfd8f2a5ab41b4c0a0fc7913649ee461R31